whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -471,11 +486,11 @@ private IpEntry lookupParentIpEntry(final IpInterval ipInterval) {
     private static Remark createRemark(final RpslObject rpslObject) {
         final List<String> descriptions = Lists.newArrayList();
 
-        for (final CIString description : rpslObject.getValuesForAttribute(AttributeType.DESCR)) {
+        for (final CIString description : rpslObject.getValuesForAttribute(DESCR)) {
```

Comment:
I suggest not removing the `AttributeType` as it makes the code less readable (i.e. I have no idea what type `DESCR` is by just reading the code).

Reply:
Done